arrowFunctions.js

```diff
@@ -4,6 +4,8 @@
  * Parameter(s):
  * nums: An array of numbers.
  */
+
+ var nums = [1, 2, 3, 4, 5];
```

Comment:
@Ayopal This part isn't necessary for the solution. It is for your personal use.

Reply:
line removed